l10n-screenshots-config.yml

```diff
@@ -0,0 +1,95 @@
+locales:
+    - af
+    - an
+    - anp
+    - ar
+    - ast
+    - az
+    - bg
+    - bn
+    - bo
+    - br
+    - bs
+    - ca
+    - co
+    - cs
+    - cy
+    - da
+    - de
+    - dsb
+    - el
+    - en-CA
+    - en-GB
+    - en-US
+    - eo
+    - es-AR
+    - es-CL
+    - es-MX
+    - es
+    - eu
+    - fa
+    - fi
+    - fr
+    - ga-IE
+    - gd
+    - gl
+    - gu-IN
+    - he
+    - hi-IN
+    - hr
+    - hsb
+    - hu
+    - hy-AM
+    - ia
+    - id
+    - is
+    - it
+    - ja
+    - jv
+    - ka
+    - kab
+    - kk
+    - km
+    - kn
+    - ko
+    - lo
+    - lt
+    - lv
+    - ml
+    - mr
+    - ms
+    - my
+    - nb-NO
+    - ne-NP
+    - nl
+    - nn-NO
+    - oc
+    - or
+    - pa-IN
+    - pl
+    - pt-BR
+    - pt-PT
+    - rm
+    - ro
+    - ru
+    - ses
+    - si
+    - sk
+    - sl
+    - sq
+    - su
+    - sv-SE
+    - ta
+    - te
+    - th
+    - tl
+    - tr
+    - uk
+    - ur
+    - uz
+    - vi
+    - zgh
+    - zh-CN
+    - zh-TW
+locales-with-shorter-context-menu: ["hi-IN", "th", "zh-CN", "zh-TW"]  # These locales have 1 less screenshot. 
```

Comment:
I think capturing edge cases by defining a list of locales that behave the same way is more maintainable. I thought of making a file that tells how many screenshots each single locale has, but that'll be cumbersome and prone to error to update.